src/vs/platform/terminal/electron-browser/localPtyService.ts

```diff
@@ -109,6 +112,10 @@ export class LocalPtyService extends Disposable implements IPtyService {
 		this._register(proxy.onProcessTitleChanged(e => this._onProcessTitleChanged.fire(e)));
 		this._register(proxy.onProcessOverrideDimensions(e => this._onProcessOverrideDimensions.fire(e)));
 		this._register(proxy.onProcessResolvedShellLaunchConfig(e => this._onProcessResolvedShellLaunchConfig.fire(e)));
+		this._register(proxy.onProcessReplay(e => {
+			this._logService.info(`Replaying ${e}`);
+			this._onProcessReplay.fire(e);
+		}));
```

Comment:
Let's avoid logging, maybe trace log in ptyService (closer to source) so we can get it if we need but it doesn't show up by default